File: src/com/bean/Contractor.java
package com.bean;

public class Contractor {
	private int userid;
	private String password;
	public Contractor(int userid, String password) {
		super();
		this.userid = userid;
		this.password = password;
	}
	public int getUserid() {
		return userid;
	}
	public void setUserid(int userid) {
		this.userid = userid;
	}
	public String getPassword() {
		return password;
	}
	public void setPassword(String password) {
		this.password = password;
	}
	@Override
	public String toString() {
		return "Contractor [userid=" + userid + ", password=" + password + "]";
	}

}

File: src/com/bean/Contractor.go
package bean

import (
	"fmt"
)

type Contractor struct {
	userid   int
	password string
}

func CreateContractor(userid int, password string) Contractor {
	return Contractor{userid: userid, password: password}
}
func (c Contractor) getUserid() int {
	return c.userid
}
func (c Contractor) setUserid(userid int) {
	c.userid = userid
}
func (c Contractor) getPassword() string {
	return c.password
}
func (c Contractor) setPassword(password string) {
	c.password = password
}
func (c Contractor) String() string {
	return fmt.Sprintf("Contractor [userid=%d, password=%s]", c.userid, c.password)
}
